__tests__/app.test.js

```diff
@@ -0,0 +1,112 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/healthcheck", () => {
+  test("it should respond with a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array);
+        body.topics.forEach((topic) => {
```

Comment:
As we're using a forEach here, we should check the length of the array beforehand.